newVersion.py

```diff
@@ -223,6 +222,22 @@ def bump_command(args):
     change_command(args)
     npm_install_dir(sdk_dirs.sdk_core)
 
+def changeall_command(args):
+    args.skip_commit_id = True
```

Comment:
I realized that I need to have it not change the iTwin Mobile SDK versions when doing a local update of iTwin. I'm going to fix that.

Reply:
Fixed.